ApprovalTests/integrations/ut/UTApprovals.h

```diff
@@ -0,0 +1,80 @@
+#ifndef APPROVALTESTS_CPP_UTAPPROVALS_H
+#define APPROVALTESTS_CPP_UTAPPROVALS_H
+
+#include "ApprovalTests/namers/ApprovalTestNamer.h"
+
+// <SingleHpp unalterable>
+#ifdef APPROVALS_UT
```

Comment:
This is all great - thank you!

I've been debating between APPROVALS_UT and  APPROVALS_BOOST_UT.

I don't have a strong opinion, but will at least write down my thoughts, to see what you think...

If we add support for boost.test, it will presumably have to be called APPROVALS_BOOST_TEST, as APPROVALS_TEST would be too ambiguous.

So that might suggest APPROVALS_BOOST_UT, for consistency.

But then there is the possibility that UT doesn't get accepted to boost - in which case the shorter name would end up better...

In general I tend to prefer longer names, if they are clearer....

What do you think?

Reply:
I was in doubt about that as well. As all the things in this boost-experimental were never officially approved into boost (even although they are high quality) I presumed this one won't either. So for the defines and files names I just stuff with UT. On the documentation text I used [Boost] UT as the brackets indicate the non official nature of the boost keyword.

Reply:
Good points. Thanks. I agree with all that, so am marking this as Resolved.